spring-data-r2dbc/src/main/java/org/springframework/data/r2dbc/mapping/R2dbcMappingContext.java

```diff
@@ -41,8 +42,10 @@ public R2dbcMappingContext() {
 	 * @param namingStrategy must not be {@literal null}.
 	 */
 	public R2dbcMappingContext(NamingStrategy namingStrategy) {
+
 		super(namingStrategy);
-		setForceQuote(false);
+
+		setForceQuote(true);
```

Comment:
`setForceQuote(…)` could be removed entirely as it sets the value to its default value, see `JdbcMappingContext`.